packages/ember-application/tests/system/logging_test.js

```diff
@@ -7,11 +7,8 @@ import View from 'ember-views/views/view';
 import Controller from 'ember-runtime/controllers/controller';
 import Route from 'ember-routing/system/route';
 import RSVP from 'ember-runtime/ext/rsvp';
-import keys from 'ember-metal/keys';
 import compile from 'ember-template-compiler/system/compile';
 
-import 'ember-routing';
```

Comment:
I believe that this is needed for side-effects.  Specifically, the run loop queues that are created come to mind (though I could be wrong).